Array/script.js

```diff
@@ -0,0 +1,28 @@
+// Create an Array #1
+var numbers = new Array()
+numbers[0] = 1
+numbers[1] = 2
+numbers[2] = 3
+numbers[3] = 4
+numbers[4] = 5
+
+// Create an Array #2
+var names = ["kaveh","kiarash"]
+
+// push it to the end of the Array
+names.push("mohammad")
+
+// Delete the last array index
+names.pop()
```

Comment:
is there /whats the return value of pop()? any difference with delete/remove method?

Reply:
pop() can delete only the last one of the Array, but when we use Delete we have to put the number of  index for example
delete names[1] , index number 1  can be deleted.